### ISSUE TITLE
html Element is missing lang attribute

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-lockbox/lockbox-extension#555]
Every HTML document should have a lang attribute. [reference](https://dequeuniversity.com/rules/axe/2.6/html-has-lang?application=axeAPI)

**Comment by linuxwolf** [comment]
@kimberlythegeek I'm not versed enough in the `lang` attribute to know how this ought to interact with localization.

@jimporter I'm guessing we insert the `lang` based on the locale we negotiate through `fluent-react`?

**Comment by sandysage** [comment]
Let's consider this a discovery effort to determine what needs to happen here and then we can prioritize the scope of work from this.
